activity/actreturn/README.md

```diff
@@ -0,0 +1,50 @@
+# tibco-reply
+This activity provides your flogo application the ability to reply to a trigger invocation.
+
+## Installation
+
+```bash
+flogo add activity github.com/TIBCOSoftware/flogo-contrib/activity/reply
+```
+
+## Schema
+Inputs and Outputs:
+
+```json
+{
+  "inputs":[
+    {
+      "name": "code",
+      "type": "integer",
+      "required": true
+    },
+    {
+      "name": "data",
+      "type": "any"
+    }
+  ],
+  "outputs": [
+  ]
+}
+```
+## Settings
+| Setting     | Description    |
+|:------------|:---------------|
+| code        | The response code |         
+| data        | The response data |
+
+## Configuration Examples
```

Comment:
Does this configuration match the current implementation? we should have the activityRef instead.